CONTRIBUTION_GUIDE.md

```diff
@@ -0,0 +1,124 @@
+# Contribution Guide
+
+---
+
+Thank you for being interested in contributing to Aragon! 🎉  We’re excited to have you built this vision with us.
+
+There are many ways to contribute, from writing tutorials or blog posts and improving the documentation, to submitting bug reports, feature requests or writing code which can be incorporated into the project.
+
+All members of our community are expected to follow our [Code of Conduct](https://discord.com/channels/672466989217873929/953988031713534022/1037740253080997918). Please make sure you are welcoming and friendly in all of our spaces. This is a non-negotiable 💙
+
+## Your first contribution
+
+Unsure where to begin contributing to Aragon?
+
+You can start with a [Good First Issue](https://github.com/aragon/aragon-apps/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+
+> Good first issues are usually for small features, additional tests, spelling / grammar fixes, formatting changes, or other clean up.
+>
+
+Start small, pick a subject you care about, are familiar with, or want to learn.
+
+If you're not already familiar with git or Github, here are a couple of friendly tutorials: [First Contributions](https://github.com/firstcontributions/first-contributions), [Open Source Guide](https://opensource.guide/), and [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+Otherwise, you can also contribute through improving our documentation. You can find our documentation within the `docs` folder of each of our repositories. If you see something that could be better explained, or if you want to share better resources for each of the topics, we’d love to hear from you. Feel free to either submit a PR directly to the repo or simply [register here](https://aragonteam.typeform.com/to/QJyKtESU) and we’ll reach out.
+
+## How to file an issue or report a bug
+
+If you see a problem, you can report it in our [issue tracker](https://github.com/aragon/aragon-apps/issues) (or [here](https://aragonteam.typeform.com/to/QJyKtESU)).
```

Comment:
This link would make sense on the App's repository, but not here IMO. 

I would link the `core` and `sdk` repositories instead

Reply:
+1! Have linked the core issues link here and will link the sdk's in that repository when we add this file there.